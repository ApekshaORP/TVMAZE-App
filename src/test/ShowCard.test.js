import React from 'react';
import ShowCard from '../component/ShowsCard';
import {shallow} from 'enzyme';
import { render, screen } from '@testing-library/react';

const SetUp=(props={})=>{
    const component=shallow(<ShowCard {...props}/>)
    return component
}
describe('Show Card component', () => {
   it ('It should render with logo',()=>{
    const component=SetUp()
    // const wrapper =component.find('.show_overview');
    // expect(wrapper.length).toBe(1);
    expect(component).toMatchSnapshot();

    
})
  });


//   test('renders home react component', () => {
//     render(<ShowCard />);
//     const linkElement = screen.getByText(/Overview/i);
//     expect(linkElement).toBeInTheDocument();
//   });