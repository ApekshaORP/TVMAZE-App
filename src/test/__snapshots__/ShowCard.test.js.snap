// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Card component It should render with logo 1`] = `ShallowWrapper {}`;
